lib/thread.c

```diff
@@ -40,6 +41,12 @@ DEFINE_MTYPE_STATIC(LIB, THREAD_STATS,  "Thread stats")
 #include <mach/mach_time.h>
 #endif
 
+#define AWAKEN(m) \
```

Comment:
This isn't used anywhere ... is this patch actually complete?

(Would've been better to put the select() removal in a separate patch, very confusing to review like this...)